package.json

```diff
@@ -115,7 +115,8 @@
     "typescript": "3.8.3"
   },
   "resolutions": {
-    "**/agent-base": "5"
+    "**/agent-base": "5",
+    "**/jest-resolve": "29.5.0"
```

Comment:
Have to bump this so that we get around https://github.com/facebook/jest/issues/9771, which requires Jest >= 28